openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -103,6 +103,18 @@
             });
 
             sectionOneFieldViews = [
+                new FieldsView.DateFieldView({
```

Comment:
This was changed in accordance with @catong 's request above: https://github.com/edx/edx-platform/pull/15843#issuecomment-323399142